driver/bpf/fillers.h

```diff
@@ -2298,6 +2298,83 @@ FILLER(sys_openat_x, true)
 	return res;
 }
 
+FILLER(sys_openat2_x, true)
+{
+	unsigned long resolve;
+	unsigned long flags;
+	unsigned long val;
+	unsigned long mode;
+	long retval;
+	int res;
+#ifdef __NR_openat2
+	struct open_how how;
+#endif
```

Comment:
Thanks for changing this. Did you get a chance to do a test build with an older (pre 5.6.0) kernel?

Reply:
Thank you!

Reply:
Thank you for your suggestions! I successfully built this on kernel version 5.4.0.